03-specifying-sample-designs.Rmd

```diff
@@ -1,50 +1,52 @@
+A sampling design is the method used to draw a sample. Both logistical and statistical elements are considered when developing a sampling design. When specifying a sampling design in R, the levels of sampling are specified along with the weights. Each record of a weight is constructed so that the particular record represents that many units in the population. For example, in a survey of 6th-grade students in the United States, the weight associated with each responding student reflects how many students that record represents. Generally, the sum of the weights sum to the population total though some studies have the sum of the weights sum to the number of respondent records. 
 
 Some common terminology across the designs are:
 
 - sample size, generally denoted as $n$, is the number of units selected
 - population size, generally denoted as $N$, is the number of units in the population
-- sampling frame is the list of units from which the sample is drawn
+- sampling frame, the list of units from which the sample is drawn
 
 ### Simple random sample without replacement
 
 - **Description**: The simple random sample (SRS) without replacement is a sampling design where a fixed sample size is selected from a sampling frame, and every possible subsample has an equal probability of selection.
 - **Requirements**: The sampling frame must include the entire population.
 - **Advantages**: SRS requires no information about the units apart from contact information.
 - **Disadvantages**: The sampling frame may not be available for the entire population. This design is not generally feasible for in-person data collection.
-- **Example**: Randomly students in a university from a roster provided by the registrar's office.
+- **Example**: Randomly students in a university from a roster provided by the registrar's office. 
 
 #### The math {-}
 
-The estimate for the population mean of variable $y$ is:
+The estimate for the population mean of variable $y$ is: 
 
 $$\bar{y}=\frac{1}{n}\sum_{i=1}^n y_i$$
 
-and the estimate of the standard error of mean is:
+and the estimate of the standard error of the mean is:
 
 $$se(\bar{y})=\sqrt{\frac{s^2}{n}\left( 1-\frac{n}{N} \right)}$$ where
 
 $$s^2=\frac{1}{n-1}\sum_{i=1}^n\left(y_i-\bar{y}\right)^2.$$
 
-This standard error estimate might look very similar to equations in other applications except for the part on the right side of the equation: $1-\frac{n}{N}$. This is called the finite population correction factor (FPC), and if the size of the frame, $N$, is very large, the FPC is negligible, so it is often ignored. 
+This standard error estimate might look very similar to equations in other applications except for the part on the right side of the equation: $1-\frac{n}{N}$. This is called the finite population correction factor (FPC), and if the size of the frame, $N$, is very large, the FPC is negligible, so it is often ignored.  
```

Comment:
Do we want to provide examples as to what the other applications that the se would look similar?

Reply:
@szimmer can you clarify where you were going for here with "other applications"?